### PR TITLE
Fix(web): Stretch input to entire container

### PR DESCRIPTION
**Pull request by literat**
Stretching input to entire container space using `width` instead of `max-width`.

Screenshot 2022-03-22 at 21 56 56

### CODE REVIEW

**Review by ondrej-kaspar — APPROVED** [review]

File: packages/web/src/tools/_forms.scss [hunk 1]
```diff
@@ -28,7 +28,7 @@
         @include typography.generate(map.get($theme, input-typography));
 
         display: block;
-        max-width: 100%;
+        width: 100%;
```
Comment (adamkudrna) [inline packages/web/src/tools/_forms.scss:31]: This would override the `size` attribute. Is it OK?

Comment (adamkudrna) [reply packages/web/src/tools/_forms.scss:31]: As discussed in person:

- `TextField` will have a default width of 288 px / 18 rem so it's in sync with Figma.
- `TextField` box model will change to `inline-block` so multiple fields can be placed in a row.
-  The `<input>` itself should be always 100% wide, it's width will be controlled by its containing `<div>`.
- There will be a new API option `isFluid` (?) for both `TextField` and `Button` so their width can be controlled by their container.
- We will postpone the support of the `size` attribute for later as it becomes a bit more difficult with the changes proposed.

**Review by adamkudrna — CHANGES_REQUESTED** [review]

File: packages/web/src/tools/_forms.scss (hunk 1 above)
Comment (literat) [reply packages/web/src/tools/_forms.scss:31]: There is a `Button-block` class to set `width: 100%` on a button. So I think that it is what we need, though.

File: packages/web/src/tools/_forms.scss [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 @mixin field($type, $variants, $theme) {
     .#{$type} {
-        max-width: 100%;
+        display: inline-block;
+        width: 18rem;
```
[the remark this answers is not in the document]
Comment (literat) [reply packages/web/src/tools/_forms.scss]: I've move it to `_theme`.

**Comment by literat** [comment]
Much better I think.

Screenshot 2022-03-25 at 15 30 59

**Review by crishpeen — APPROVED** [review]

**Review by adamkudrna — APPROVED** [review]

**Pull request merged**